src/components/ScholarshipDetailCard.jsx

```diff
@@ -208,6 +211,11 @@ export default function ScholarshipDetailCard({ scholarship }) {
       </Box>
       <Chip
         component={Link}
+        href={genMailToLink({
+          subject: `Report Issue for ${name}`,
+          bcc: authorEmail,
+          body: `Please describe the issue for the scholarship located at https://${window.location.hostname}${location.pathname}:`,
```

Comment:
Two things:
1. The URL also appears on line 78. Maybe refactor it?
2. Diagnostic info is probably helpful here too.

Reply:
got it, I also removed `useLocation` since it wasn't really needed